ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -25,56 +27,67 @@ function createMarkdownPanelStyles(theme: Theme) {
     // Make the content scrollable
     height: '100%',
     overflowY: 'auto',
-    // Ignore first margin
+    // Ignore initial margin
     '& :first-child': {
       marginTop: 0,
     },
-    // Styles for <code>
+    // <code>
     '& code': { fontSize: '0.85em' },
     '& :not(pre) code': {
       padding: '0.2em 0.4em',
-      backgroundColor: theme.palette.grey[100],
+      backgroundColor: theme.palette.mode === 'light' ? '#f5f5f5' : '#2c2c2c',
       borderRadius: '4px',
     },
     '& pre': {
       padding: '1.2em',
-      backgroundColor: theme.palette.grey[100],
+      backgroundColor: theme.palette.mode === 'light' ? '#f5f5f5' : '#2c2c2c',
```

Comment:
Isn't a better idea to use the color from the panelette ? Or if the color are not well defined, to add these colors in the palette.
My little knowledge around theming is we shouldn't have hardcoded color, because you have theme that defined for your the color you have to used.
Quite useful when you want to swap from a theme to another. Here for example from light to dark mode. But we could have multiple different dark or light themes with different color contrast. That would be useful for the accessibility for example.

Reply:
👍  to this--should try to use theme tokens here rather than hardcoding colors if possible so that consumers have control over the color via the theme. Totally cool to keep the conditional for light vs dark though.